.github/workflows/pip_installation.yml

```diff
@@ -9,7 +9,7 @@ on:
   push:
     branches: [ main, development ]
   pull_request:
-    branches: [ main, development ]
+    branches: [ main ]
```

Comment:
We don't need the tests on PR merging into development?

Reply:
I would suggest running tests only on 1 os,, cf. https://github.com/MannLabs/alphabase/blob/284b2f6241a214247f39a446b3e0f330367a8047/.github/workflows/pip_installation.yml#L3C1-L6C1

Reply:
I see.